dev/PowerNotifications/PowerNotifications.h

```diff
@@ -649,14 +604,16 @@ namespace winrt::Microsoft::ProjectReunion::factory_implementation
         }
 
     };
-
 };
 
 namespace winrt::Microsoft::ProjectReunion::implementation
 {
+    // EffectivePowerMode variables
+    static ULONG g_powerModeVersion = NULL;
```

Comment:
Use `0` instead of `NULL` here, this is a ULONG not a pointer.

Reply:
Of course I would use a `nullptr` instead if it was a pointer. But I prefer using NULL. Is there a guidance against it?

Reply:
Yes, because `NULL` is for pointers.  If you want an "unknown yet" model use `std::optional<ULONG>` instead. Don't use `NULL` here. If you mean zero, say literal zero.